data/categories.json

```diff
@@ -60,6 +60,12 @@
     "title": "Tag Management",
     "description": "These scripts tend to load lots of other scripts and initiate many tasks."
   },
+  "cmp": {
```

Comment:
WDYT about `consent-manager`? CMP pulls double duty in other tech subcircles

Reply:
Yes, think that probably makes more sense - explicit is generally better